.github/workflows/e2e.yaml

```
@@ -73,7 +76,7 @@ jobs:
       - if: always() && inputs.event_name == 'workflow_run'
         uses: ./.github/actions/commit-status/start
         with:
-          name: "${{ github.workflow }} / e2e (${{ inputs.suite }}) / ${{ github.job }} (snapshot)"
+          name: ${{ github.workflow }} (${{ inputs.k8s_version }} ) / e2e (${{ inputs.suite }})
```

```suggestion
          name: ${{ github.workflow }} (${{ inputs.k8s_version }}) / e2e (${{ inputs.suite }})
```

There's an extra space here